### ISSUE TITLE
colmap and images workflow? 

### ISSUE THREAD

**Issue by jaggzh** [nv-tlabs/FlexiCubes#6]
Is there a way to go from photos, processed with colmap, and a hand-crafted mesh (from a poisson reconstruction), to refine the mesh to match the photos?

**Comment by frankshen07** [comment]
Yes, you can check out nvdiffrec, which now integrates FlexiCubes as well.
